src/software/simulated_tests/validation_functions/enemy_defense_ball_touch_validation.cpp

```diff
@@ -0,0 +1,24 @@
+#include "software/simulated_tests/validation_functions/enemy_defense_ball_touch_validation.h"
+
+void friendlyTouchedBallInEnemyDefense(std::shared_ptr<World> world_ptr,
+                                       ValidationCoroutine::push_type& yield)
+{
+    auto friendly_touched_ball = [](std::shared_ptr<World> world_ptr) {
+        Point ball_position = world_ptr->ball().position();
+        for (const auto& robot : world_ptr->friendlyTeam().getAllRobots())
+        {
+            robot.double abs_error =
+                std::fabs((robot.position() - ball_position).length() - 1.0);
+            if (abs_error > 0.05)
```

Comment:
Same question about where 0.05 came from. Commenting would be helpful here.

Reply:
Also need to take radii into account here